starting-oss-projects.md

```diff
@@ -1,34 +1,155 @@
+* Is there enough external interest to form and grow a developer community?
+
+(Source: [Linux Foundation](https://www.linuxfoundation.org/en/resources/open-source-guides/starting-an-open-source-project/))
+
+In addition, the following aspects should be considered in the planning phase:
+
+* What is the goal of the project and will it solve the problem?
+* Are there enough resources not only to start, but to support the project in the long-term? (You also need to obtain and ensure sponsorship)
+* Does it make sense to donate the code to a vendor-neutral, non-profit organization (that is, an open source foundation), or retain some control by owning and running the project under the responsibility of your company? Note that this decision depends on the project and may also be taken later in the life cycle. Typically, a project first needs to be published and generate interest in the community before it is handed over to a third-party organization.
+* Launch check list. For example, see the [checklist of the Linux Foundation](https://www.linuxfoundation.org/en/resources/open-source-guides/starting-an-open-source-project/#checklist)
+* Set up an open source project governance. It establishes how to contribute to or maintain a project.
+* Determine the tools and infrastructure the project members will use
+* Carry out a technical review
+
+  * Ensure that all critical content is removed from the project before publishing it. For example:
+
+    * Dependencies to non-public components
+    * Internal comments, references to other internal code, and the like
+    * Access tokens and the like
+
+  * Ensure coding style is consistent
+
+* Where will the code be published? Typically, it will be in a company-owned organization on a code hosting platform auch as Githuib.com or GitLab.com but, depending on the technology, other potential publishing channels exist (for example, NPM, Maven central, PyPI)
+* Does it make sense to publish binaries? If yes, where?
+* Define your Web site and communication: What can you do to make your project known? Does it make sense to create a Web site for the project? Are there working groups?
+* Plan your project life cycle
+
+#### Different Project Levels
+
+It can make sense to have different levels for new open source projects ("sandbox", "incubator", "graduated" - these are the different [project levels of CNCF](https://www.cncf.io/projects/), for example). That's a way to classify you open source projects wrt. adoption, maturity and quality criteria that they have to fulfill. The basic idea is that new projects start in a dedicated space (CNCF calls that "sandbox" - at Facebook, that's the ["Incubator"](https://github.com/facebookincubator)). In this space, projects can evolve, and check if they reach the goals that have been defined in terms of adoption and quality. If they do, they can be promoted to the next level. If they don't it might be decided to sunset them.
```

Comment:
Is this something you would also do for projects run under governance of one company, or is this only applicable for projects in the bigger foundations which provide some framework for that?

Reply:
This is also something for projects that are under the responsibility of only one company. Some companies (Facebook is mentioned) do this already.